src/modules/options/defaultConfig.js

```diff
@@ -53,3 +53,50 @@ export default ({
     ]),
     getLimitValuesTab(),
 ]
+
+export const defaultOptionNames = ({
+    isStacked,
+    supportsLegends,
+    isColumnBased,
+}) => {
+    const options = [
+        'cumulativeValues',
+        'hideEmptyRowItems',
+        'sortOrder',
+        'skipRounding',
+
```

Comment:
Are the empty lines added to separate between options going on different tabs in the ui? Maybe a comment instead of empty line would be more informative.